### DISCUSSION TITLE
github copilot is  not  working

### DISCUSSION THREAD

**Discussion by bi305** [community/community#7738] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

**Comment by davecheney** [comment, marked as answer]
This error indicates that you are either not part of the beta invite list, or there was an error connecting to api.github.com to request a token to use the Copilot service. Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by bi305** [reply]
![image](https://user-images.githubusercontent.com/72016671/141889939-49c27115-ab99-4358-ad08-da3c9e7f9f42.png)
 I am in beta  list  but  still getting  this token error

**Reply by bi305** [reply]
![image](https://user-images.githubusercontent.com/72016671/141890702-f4668770-400d-489f-b3e5-01365a082148.png)

**Reply by davecheney** [reply]
Is your clock correct?

**Reply by bi305** [reply]
yes I  think  so  it  is

**Reply by bi305** [reply]
can you  please help me  to  activate  my  github copilot

**Reply by davecheney** [reply]
I have confirmed that your invite is active. Please check the output of view -> output; github copilot, the problem is most likely on your laptop or local network.

**Reply by bi305** [reply]
its working  now  thanks  so  much I appreciate

**Reply by Alii-isk** [reply]
> Is your clock correct?

i adjusted my clock and time zone and now copilot is working again ,thanks

**Comment by etemtolu** [comment]
GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"
I am also getting such an error. Can you help me too?
My invitation is available but not confirmed.

**Reply by davecheney** [reply]
@etemtolu you are on the waitlist. We are inviting people as we have capacity. Thank you for your patience and understanding.

**Comment by deepak-likes-code** [comment]
Hey, even I'm having the same error. I've the invite through mail for the technical preview but I'm getting this error 

[INFO] [auth] [2021-11-24T03:13:03.612Z] Invalid copilot token: invalid expires_at:1637703803 now:1637723583
[ERROR] [default] [2021-11-24T03:13:03.613Z] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

**Reply by davecheney** [reply]
is the clock on your computer set correctly?

**Comment by Senlisz** [comment]
GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token" ?
